test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/ValueFormTests/FirstLowerCaseValueFormTests.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.TemplateEngine.Orchestrator.RunnableProjects.ValueForms;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests.ValueFormTests
+{
+    public class FirstLowerCaseValueFormTests
+    {
+        [Theory]
+        [InlineData("A", "a")]
+        [InlineData("NO", "nO")]
+        [InlineData("NEW", "nEW")]
+        [InlineData("", "")]
+        [InlineData(null, null)]
```

Comment:
Regardless of what culture related decision we make, I think we should add tests for different cultures.